### DISCUSSION TITLE
trying to run the scan_for.sh

### DISCUSSION THREAD

**Discussion by clint70** [jekhokie/raspberry-noaa-v2#555] [Q&A]
Ran the scan_for.sh 4h and no output?  the antenna is working, when following the trouble shooting and running the test_reception.sh XXX.XX (freq) that works, audio and strong.  I ran the scan_for.sh again with 4m NOTHING again, so ran and then used the command 
`./scripts/testing/stop_and_finalize_scanning.sh /home/pi/raspberry-noaa-v2/tmp/scanner/timed_scan_24.08.22-09.06.csv.gz`

with the resulting errors:

```generating /home/pi/raspberry-noaa-v2/tmp/scanner/timed_scan_24.08.22-09.06.csv.gz.png from /home/pi/raspberry-noaa-v2/tmp/scanner/timed_scan_24.08.22-09.06.csv.gz.
This may take a while, maybe get a coffee?
loading
Traceback (most recent call last):
  File "/home/pi/raspberry-noaa-v2/scripts/testing/heatmap.py", line 155, in gzip_wrap
    line = next(iterator)
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pi/raspberry-noaa-v2/scripts/testing/heatmap.py", line 612, in <module>
    summarize_pass(args)
  File "/home/pi/raspberry-noaa-v2/scripts/testing/heatmap.py", line 233, in summarize_pass
    for line in raw_data():
RuntimeError: generator raised StopIteration
done.```

**Comment by cjuan** [comment]
I was the same problem, but I fixed with this:
Modify file scripts/testing/heatmap.py and change line 159:
`except IOError:`
for this one:
`except (EOFError, StopIteration):`

This solution was good for me.

**Reply by clint70** [reply]
I will try... thank you.  Looks like I was running an older ver. too, so updating to the newer MeteorDemod. BUT Will try to fix this too.
